sample/build.gradle

```diff
@@ -158,8 +158,8 @@ android {
 
 dependencies { configuration ->
 
-    releaseTestKujakuImport(configuration)
-    //developmentKujakuModulesImport(this, configuration)
+    //releaseTestKujakuImport(configuration)
+    developmentKujakuModulesImport(this, configuration)
```

Comment:
Also update the mapbox SDK version to the effective one which is this

![Screen Shot 2019-07-17 at 1 40 20 PM](https://user-images.githubusercontent.com/31766075/61369335-89afb580-a898-11e9-9317-1b65b721f250.png)

This makes it easier for someone to know that that's the version of Mapbox being used instead of the one stated in the build.gradle constants

Reply:
Hi, do you want me to change the "mapboxSdkVersion" from 7.2.0 to 7.3.0 in the global build.gradle ?

Reply:
Yes, and wherever else it's declared statically